scripts/scraper-ng/preset.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
     [require("./plugins/load-recipes")],
     [require("./rules/html-no-macros"), allowedMacros],
     [require("./rules/html-require-recipe-ingredients")],
+    [require("./rules/html-require-ingredient-order")],
```

Comment:
We should have a comment here that this line must come after the "html-require-recipe-ingredients" one.

Reply:
Done with bf3406eceddeb07b8af546fc34df8b204ffed80b